.github/workflows/test.yml

```diff
@@ -79,7 +79,12 @@ jobs:
           terraform_version: ${{ matrix.terraform }}
```

Comment:
lets also configure this ci test to run weekly so that we know if anything breaks